### ISSUE TITLE
Ummm

### ISSUE THREAD

**Issue by ZekePolarisBSH** [giolaq/helloflipper#2]
Should it close the moment I press the middle button? And how does an airplane count as "Hello Flipper"?

**Comment by MorevPro** [comment]
Yes, the default operator is responsible for ending the while loop if another value of event.key is received that does not match the values of InputKeyLeft, InputKeyRight, InputKeyUp, or InputKeyDown. This is done by setting the running variable to false, which will result in exiting the while loop.

` default:
                    running = false;
                    break;
                }`

**Comment by ZekePolarisBSH** [comment]
So turn it to false and save and try again or just don't use it correct?

**Comment by MorevPro** [comment]
I would recommend replacing the code:

```
default:
    running = false;
    break;
```


with:

```
case InputKeyBack:
    running = false;
    break;
default:
    break;
```

**Comment by ZekePolarisBSH** [comment]
I'm replying via my phone using email right now, ah ok thank you. I will
close this in a bit. I don't know how to do it from here if there is a way.
Thanks again
